src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/win/Microsoft.NETCore.Runtime.CoreCLR.pkgproj

```diff
@@ -9,19 +9,20 @@
     <PackageTargetRuntime>$(MinOSForArch)-$(PackagePlatform)</PackageTargetRuntime>
   </PropertyGroup>
   <ItemGroup>
-    <ArchitectureSpecificNativeFile Include="$(BinDir)clretwrc.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)coreclr.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)dbgshim.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)mscordaccore.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)mscordbi.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)clretwrc.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)coreclr.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)dbgshim.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)mscordaccore.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)mscordbi.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)mscorrc.debug.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)mscorrc.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)sos.dll" />
     <ArchitectureSpecificNativeFile Include="$(BinDir)mscorlib.ni.dll" />
     <ArchitectureSpecificNativeFile Include="$(BinDir)System.Private.CoreLib.ni.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)mscorrc.debug.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)mscorrc.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)sos.dll" />
     <ArchitectureSpecificLibFile Include="$(BinDir)System.Private.CoreLib.dll" />
     <ArchitectureSpecificLibFile Include="$(BinDir)mscorlib.dll" />
```

Comment:
The pdb’s for mscorlib.dll and System.Private.CoreLib.dll are included in the symbol package.

Archive:  runtime.win7-x64.microsoft.netcore.runtime.coreclr.1.0.3-rc4-24219-0.symbols.nupkg
  Length      Date    Time    Name
---------  ---------- -----   ----
      544  06/20/2016 16:01   _rels/.rels
     1271  06/20/2016 16:01   runtime.win7-x64.Microsoft.NETCore.Runtime.CoreCLR.nuspec
     9338  06/20/2016 16:01   dotnet_library_license.txt
     1617  06/20/2016 16:01   ThirdPartyNotices.txt
        0  06/20/2016 16:01   ref/netstandard1.0/_._
        0  06/20/2016 16:01   runtimes/win7-x64-aot/lib/netstandard1.0/_._
        0  06/20/2016 16:01   runtimes/win7-x64-aot/native/_._
    86016  06/20/2016 16:01   runtimes/win7-x64/native/clretwrc.pdb
96964608  06/20/2016 16:01   runtimes/win7-x64/native/coreclr.pdb
11849728  06/20/2016 16:01   runtimes/win7-x64/native/dbgshim.pdb
37548032  06/20/2016 16:01   runtimes/win7-x64/native/mscordaccore.pdb
24547328  06/20/2016 16:01   runtimes/win7-x64/native/mscordbi.pdb
    86016  06/20/2016 16:01   runtimes/win7-x64/native/mscorrc.debug.pdb
    86016  06/20/2016 16:01   runtimes/win7-x64/native/mscorrc.pdb
14667776  06/20/2016 16:01   runtimes/win7-x64/native/sos.pdb
  6192640  06/20/2016 16:01   runtimes/win7-x64/native/System.Private.CoreLib.pdb
     7680  06/20/2016 16:01   runtimes/win7-x64/native/mscorlib.pdb
40079360  06/20/2016 16:01   runtimes/win7-x64/native/crossgen.pdb
        0  06/20/2016 16:01   src/_._
      589  06/20/2016 16:01   [Content_Types].xml
     1017  06/20/2016 16:01   package/services/metadata/core-properties/846fbb9ad70743ecb6a98cdae9d0b5c6.psmdcp
---------                     -------
232129576                     21 files

It looks like they are also included in the binary package:

C:\ssd\coreclr2\bin\Product\Windows_NT.x64.Debug\.nuget\pkg>unzip -l runtime.win7-x64.microsoft.netcore.runtime.coreclr.1.0.3-rc4-24219-0.nupkg
Archive:  runtime.win7-x64.microsoft.netcore.runtime.coreclr.1.0.3-rc4-24219-0.nupkg
  Length      Date    Time    Name
---------  ---------- -----   ----
      544  06/20/2016 16:01   _rels/.rels
     1271  06/20/2016 16:01   runtime.win7-x64.Microsoft.NETCore.Runtime.CoreCLR.nuspec
     9338  06/20/2016 16:01   dotnet_library_license.txt
     1617  06/20/2016 16:01   ThirdPartyNotices.txt
        0  06/20/2016 16:01   ref/netstandard1.0/_._
        0  06/20/2016 16:01   runtimes/win7-x64-aot/lib/netstandard1.0/_._
        0  06/20/2016 16:01   runtimes/win7-x64-aot/native/_._
  2476544  06/20/2016 16:01   runtimes/win7-x64/lib/netstandard1.0/System.Private.CoreLib.dll
    30720  06/20/2016 16:01   runtimes/win7-x64/lib/netstandard1.0/mscorlib.dll
    33280  06/20/2016 16:01   runtimes/win7-x64/native/mscorlib.ni.dll
17038336  06/20/2016 16:01   runtimes/win7-x64/native/System.Private.CoreLib.ni.dll
   223232  06/20/2016 16:01   runtimes/win7-x64/native/clretwrc.dll
22261760  06/20/2016 16:01   runtimes/win7-x64/native/coreclr.dll
  2003968  06/20/2016 16:01   runtimes/win7-x64/native/dbgshim.dll
  5150720  06/20/2016 16:01   runtimes/win7-x64/native/mscordaccore.dll
  5206528  06/20/2016 16:01   runtimes/win7-x64/native/mscordbi.dll
   355328  06/20/2016 16:01   runtimes/win7-x64/native/mscorrc.debug.dll
     4608  06/20/2016 16:01   runtimes/win7-x64/native/mscorrc.dll
  2358272  06/20/2016 16:01   runtimes/win7-x64/native/sos.dll
  7941120  06/20/2016 16:01   tools/crossgen.exe
      652  06/20/2016 16:01   [Content_Types].xml
     1017  06/20/2016 16:01   package/services/metadata/core-properties/2209d02668094232a8db503ecb20a5a3.psmdcp
---------                     -------
65098855                     22 files